javascript-1/script3.js

```diff
@@ -0,0 +1,24 @@
+
+console.log("EXERCITIUL 3");
+
+let luni=0;
```

Comment:
La exercitiul acesta nu ai folosit variabilele deloc, as fi vrut sa le afisezi in dreptul la fiecare zi a saptamanii.
```js
console.log(`Marti ${marti}`);
```